Backpack/Button/Classes/BPKButton.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+/**
+ * Enum values for specifying button size
+ */
+typedef NS_ENUM(NSUInteger, BPKButtonSize) {
+    BPKButtonSizeDefault = 0,
+    BPKButtonSizeLarge = 1,
+};
+
+/**
+ * Enum values for specifying button style
+ */
+typedef NS_ENUM(NSUInteger, BPKButtonStyle) {
+    BPKButtonStylePrimary = 0,
+    BPKButtonStyleSecondary = 1,
+    BPKButtonStyleDestructive = 2,
+    BPKButtonStyleFeatured = 3,
+};
+
+/**
+ * Enum values for specifying image position compared to the title label
+ */
+typedef NS_ENUM(NSUInteger, BPKButtonImagePosition) {
+    BPKButtonImagePositionRight = 0,
```

Comment:
Can we rename these to `leading` and `trailing`?

Reply:
Sure, good point!